tools/db_crashtest.py

```diff
@@ -155,7 +155,7 @@
         [t * 16384 if t < 3 else 1024 * 1024 * 1024 for t in range(1, 30)]
     ),
     # Sync mode might make test runs slower so running it in a smaller chance
-    "sync": lambda: random.choice([1 if t == 0 else 0 for t in range(0, 20)]),
```

Comment:
Is this change needed?

Reply:
Temporary, for testing the write fault injection.